README.md

```diff
@@ -107,6 +107,24 @@ job.batch/volcano-admission-init   1/1           48s        96s
 
 ```
 
+### Install via helm
+
+To install official release, please visit to https://github.com/volcano-sh/helm-charts for details.
```

Comment:
Is it better to update `https://github.com/volcano-sh/helm-charts` to [helm-charts](`https://github.com/volcano-sh/helm-charts`)

Reply:
Done.